src/components/subscriptions/SubscriptionDetailContextProvider.jsx

```diff
@@ -34,10 +34,15 @@ const SubscriptionDetailContextProvider = ({ children, subscription }) => {
     userStatusFilter,
   });
 
+  const forceRefreshDetailView = () => {
```

Comment:
Should we assert the refresh calls are made properly in our tests since it is core to the experience? (seeing codecov comments about uncovered lines about the refresh stuff in this PR)?